js/script.js

```diff
@@ -0,0 +1,222 @@
+// All the DOM selectors stored as short variables 
```

Comment:
You could write the DOM selectors like this instead, to make the code shorter. 
![Skärmavbild 2022-02-07 kl  12 08 15](https://user-images.githubusercontent.com/54495430/152777111-bcf37b1e-4238-4788-9323-85eb86ae1382.png)

Reply:
Thanks